public_site/index.js

```diff
@@ -0,0 +1,16 @@
+let OSName = "Unknown";
+let OSDownloadMap = {
+    "Windows":"https://github.com/lockbook/lockbook/blob/master/docs/guides/install/windows.md",
```

Comment:
There's no way to have it link to the installer for the latest release is there? 

kinda want windows to go here: https://github.com/lockbook/lockbook/releases

Reply:
There's a way to make the download button fetch the latest windows installer and automatically download the binary. This can be achieved through GitHub REST API. 
1. [get the assets of the latest release](https://docs.github.com/en/rest/releases/releases?apiVersion=2022-11-28#get-the-latest-release)
2. get the `browser_download_url` of the windows installer: [example](https://api.github.com/repos/lockbook/lockbook/releases/assets/82753551) 

Do you want to move in this direction? This approach would be ideal UX as it reduces the steps a windows user takes to install the binary.

Reply:
yeah that's a reasonable end state, but for now maybe we'll just hard code the windows release link, and we can have a release checklist. Solve the problem using organizational operations rather than code for now

Reply:
sort that out and I'll merge this